main/uis/ButtonGroup/__tests__/ButtonGroup.test.tsx

```diff
@@ -171,6 +191,51 @@ describe('[ButtonGroup]', () => {
 
       expect(selectedItemCount).toBe(1);
     });
+
+    it('should item has given styles', () => {
+      const selectedIndex = 0;
+
+      const data: string[] = ['Item1', 'Item2', 'Item3'];
+
+      const styles = {
+        selectedButton: {
+          backgroundColor: 'red',
+        },
+        selectedText: {
+          color: 'white',
+        },
+        unselectedButton: {
+          backgroundColor: 'pink',
+        },
+        unselectedText: {
+          color: 'black',
+        },
```

Comment:
Make it a single line with single property.
https://github.com/hyochan/style-guide/blob/main/docs/ko/REACT.md#3-5-9-%ED%95%98%EB%82%98%EC%9D%98-%ED%94%84%EB%A1%9C%ED%8D%BC%ED%8B%B0%EB%8A%94-%ED%95%9C%EC%A4%84%EB%A1%9C-%EB%A7%8C%EB%93%A4%EA%B8%B0